ios/FluentUI.Demo/FluentUI.Demo/Demos/PopupMenuDemoController.swift

```diff
@@ -46,6 +46,11 @@ class PopupMenuDemoController: DemoController {
         navigationController?.isToolbarHidden = false
     }
 
+    override func viewWillDisappear(_ animated: Bool) {
```

Comment:
Just as sanity checks:
Have you validated it on iOS 14 & 15, different iPhone and iPad models?
(please update the checkmarks at the bottom of the description)

Reply:
Yes, all works! Updated description